cypress/fixtures/pages.json

```diff
@@ -0,0 +1,20 @@
+{
+    "registerPage": {
+      "url": "https://conduit.realworld.how/register",
```

Comment:
@karenn32 Those are not the values to add to the fixture files, add here the values to assert, to add and so on. Especially with cypress, be careful with that.